app/src/main/res/layout/fragment_address_editor.xml

```diff
@@ -487,7 +477,7 @@
             android:paddingEnd="12dp"
             android:text="@string/addresses_save_button"
             app:layout_constraintTop_toTopOf="@+id/cancel_button"
-            app:layout_constraintBottom_toBottomOf="@+id/cancel_button"
+            app:layout_constraintBottom_toBottomOf="@id/cancel_button"
```

Comment:
follow-up: I don't believe the text buttons don't have the click ripple on them.